restcomm/restcomm.mgcp/src/main/java/org/restcomm/connect/mgcp/IvrEndpoint.java

```diff
@@ -119,7 +119,7 @@ private void stop(Object message) {
         request.setNotifiedEntity(agent);
 
 //        //Ask RMS to notify for Recording Signal
-        if (se.getEvent().getName().equalsIgnoreCase("pr")) {
+        if (se.getEvent().getName().equalsIgnoreCase("pr") || se.getEvent().getName().equalsIgnoreCase("pa") ) {
```

Comment:
ummm, such nice short easy fix, just supportign to request a new type of event "pa",right?

could we add a meaningfull comment (maybe reuse the previous one augmented " //Ask RMS to notify for Recording Signal") to explain what we fixed here (ticket url and brief explanation)?

Reply:
sure @jaimecasero , makes sense